openapi-maven-plugin/src/main/java/io/github/kbuntrock/model/Endpoint.java

```diff
@@ -96,4 +97,13 @@ public boolean isDeprecated() {
 	public void setDeprecated(final boolean deprecated) {
 		this.deprecated = deprecated;
 	}
+
+	@Override
+	public int compareTo(final Endpoint o) {
+		return Comparator
+			.comparing((Endpoint endpoint) -> endpoint.path, nullsLast(String::compareTo))
+			.thenComparing(e -> e.type.getSchemaOrder())
```

Comment:
```suggestion
			.thenComparing(e:getType)
```

Enums are comparable by default

Reply:
Oh, nice, removed